<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaszczyznaKrawedzWindow</class>
 <widget class="QWidget" name="PlaszczyznaKrawedzWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1121</width>
     <height>551</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>201</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Płaszczyzna</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Krawędź</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Dalej</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>771</width>
       <height>391</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>401</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Głębokość elementu obrabianego:        </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_plaszczyznaGlebokosc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Szerokość całkowita (parametr):           </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_plaszczyznaSzerokosc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Długość całkowita (parametr):              </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_plaszczyznaDlugosc"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>531</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>401</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Głębokość elementu obrabianego:        </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_krawedzGlebokosc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Szerokość całkowita (parametr):           </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_krawedzSzerokosc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Długość całkowita (parametr):              </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_krawedzDlugosc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Promień naroża:                                    </string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>541</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
